tools/ci_build/github/linux/docker/scripts/install_onnx.sh

```diff
@@ -24,7 +24,9 @@ else
    PYTHON_EXE="/usr/bin/python${PYTHON_VER}"
 fi
 
-${PYTHON_EXE} -m pip install protobuf
```

Comment:
Nit: ‘pin’. 
Nit: I guess it is true not just for python 3.5 but for 3.5+ as the enforcement logic in the new typing package only allows the package to be installed on py2.7 and py3.4